builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -171,6 +171,10 @@ func resourceAwsRoute53ZoneRead(d *schema.ResourceData, meta interface{}) error
 		d.Set("delegation_set_id", cleanDelegationSetId(*zone.DelegationSet.Id))
 	}
 
+	if zone.HostedZone != nil && zone.HostedZone.Config != nil && zone.HostedZone.Config.Comment != nil {
```

Comment:
I'm not entirely familiar with what the underlying API does here but it could actually be important as it means that unsetting a comment in the UI will not be reflected in the state if it becomes nil vs empty string? (Also not sure whether removing a comment is even a thing, not overly familiar with R53)